datajunction-server/datajunction_server/api/namespaces/methods.py

```diff
@@ -0,0 +1,111 @@
+"""
+Helper methods for namespaces endpoints.
+"""
+from datetime import datetime
+from typing import List
+
+from sqlalchemy import and_
+from sqlalchemy.sql.operators import is_
+from sqlmodel import Session, select
+
+from datajunction_server.models import History
+from datajunction_server.models.history import ActivityType, EntityType
+from datajunction_server.models.node import Node, NodeNamespace, NodeType
+from datajunction_server.typing import UTCDatetime
+
+
+def get_nodes_in_namespace(
+    session: Session,
+    namespace: str,
+    node_type: NodeType = None,
+    deactivated: bool = False,
+) -> List[str]:
+    """
+    Gets a list of node names in the namespace
+    """
+    where_clause = (
+        and_(
+            Node.namespace.like(  # type: ignore  # pylint: disable=no-member
+                f"{namespace}%",
+            ),
+            Node.type == node_type,
+        )
+        if node_type
+        else Node.namespace.like(  # type: ignore  # pylint: disable=no-member
+            f"{namespace}%",
+        )
+    )
+
+    list_nodes_query = select(Node.name).where(
+        where_clause,
+    )  # .where(is_(Node.deactivated_at, None))
```

Comment:
This doesn't seem finished. I think you need to fix it for `deactivated = True`.

Reply:
Makes sense, I changed the name to `include_deactivated`